.simplecov

```diff
@@ -2,6 +2,9 @@ SimpleCov.start do
   track_files 'lib/**/*.rb'
   add_filter 'lib/reek/version.rb' # version.rb is loaded too early to test
   add_filter 'lib/reek/cli/options.rb' # tested mostly via integration tests
+  add_filter 'spec/'
+  add_filter 'samples/'
+  coverage_dir 'tmp/coverage'
```

Comment:
Do you prefer this over just `coverage/`?

Reply:
I never look at the report unless it breaks, and even then the console output is helpful enough to adress the problem. Having "coverage" at the root level makes it seems very important which it isnt, at least for me. If you feel differently i am happy to change it back.

Reply:
Makes sense :+1: